opencti-platform/opencti-front/src/private/components/locations/regions/RegionPopover.tsx

```diff
@@ -77,6 +80,10 @@ const RegionPopover = ({ id }: { id: string }) => {
         handleClose();
         navigate('/dashboard/locations/regions');
       },
+      onError: (error) => {
+        const { errors } = (error as unknown as RelayError).res;
+        MESSAGING$.notifyError(errors[0].message);
+      },
```

Comment:
Is it on purpose we override the default onError from useApiMutation ?

Reply:
Sorry, I forgot to remove that.
Fixed